guide/src/dev/inline-processing.md

```diff
@@ -0,0 +1,149 @@
+# Inline Processing
+
+The second pass of pulldown-cmark's parsing process handles inline elements like emphasis, links, and code spans.
```

Comment:
Forgot this, for completeness
```suggestion
The second pass of pulldown-cmark's parsing process handles inline elements like emphasis, links, and code spans.

Corresponds to Phase 2 of ["A Parsing Strategy"](https://spec.commonmark.org/0.31.2/#appendix-a-parsing-strategy).
```

Reply:
Actually, we don't really quite use the described algorithms for phase 2, even though it's conceptually doing the same things. I'll just close this thread